components/youtube_analytics_api/actions/common/reports-query.mjs

```
@@ -104,14 +80,10 @@
 
       return utils.arrayToCommaSeparatedList(
-        Object.entries(filtersObj)
-          .reduce((acc, [
-            key,
-            val,
-          ]) => [
-            ...acc,
-            `${key}==${val}`,
-          ], []),
+        Object.entries(filtersObj).reduce(
+          (acc, [key, val]) => [...acc, `${key}==${val}`],
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)